setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import find_packages, setup
 
 requirements = [
-    "cachetools~=4.2.1",
+    "cachetools~=5.3.0",
```

Comment:
We can make this a bit more permissive if we express the dependency requirement as a range. That way if someone really wants to run version 4, they can still opt in to doing that.

```suggestion
    "cachetools>=4.2.1,<6"
```